types/video.js/dist/video.min.d.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * This provides `video.min.js` alternative distribution typings :
+ * - `import videojs from 'video.js/dist/video.min.js';`
+ * - `import('video.js/dist/video.min.js').then((videoJSBundle) => {})`
+ */
+import videojs from '../index';
+export default videojs;
```

Comment:
should it have at least `export as namespace ....`, as this is kind of an UMD module.

Reply:
Agreed